app/components/map-from-id.js

```diff
@@ -23,6 +24,15 @@ export default Component.extend({
     closeOnClick: false,
   }),
 
+  @computed('mapConfig', 'toggledGeographyLevel')
+  geographyLevel(defaultGeographyLevel = 'county', toggledGeographyLevel = null) {
```

Comment:
`defaultGeographyLevel` seems to be an object, so its default shouldn't be a string. Maybe you meant for `toggledGeographyLevel` to default to county?